### DISCUSSION TITLE
Restreaming using Streamlink

### DISCUSSION THREAD

**Discussion by ZeroEnna** [streamlink/streamlink#3592] [General]
Hey, just a quick question: Is it possible to restream twitch streams using stream link?
The reason for my question is that I personally find the "Autohost" feature on twitch quite useless. Yeah, you restream or "host" other channels, but your OWN followers are not notified when your stream goes up for a host. Thus my idea to restream other streams. My problem is that I don't have the bucks to keep my system running 24/7, so I thought I may setup a virtual server running streamlink, and remote control that server to start/stop the restreams.

Thank you in advance.

Kind Regards

Zero

**Comment by gravyboat** [comment]
> Is it possible to restream twitch streams using stream link?

People have built apps around doing so. While it's possible via options like [--player-external-http](https://streamlink.github.io/cli.html#cmdoption-player-external-http) it's not something we recommend or support. Streamlink is designed for personal viewing, not for rebroadcasting. As someone who has been using Twitch for a while it seems unnecessary. You'd be better off looking at the API documents and querying the API to see if they're live and it's a host or not.

**Reply by ZeroEnna** [reply]
Thanks for your input. After some fumbling, I manage to get it working by piping the output to ffmpeg, and stream it from there.

**Reply by beardypig** [reply]
@ZeroEnna that would be the best method.
